scripts/ci/pre_commit/compile_fab_assets.py

```diff
@@ -78,10 +78,10 @@ def compile_assets(www_directory: Path, www_hash_file_name: str):
             cwd=os.fspath(www_directory),
             text=True,
             check=False,
-            # capture_output=True,
+            capture_output=True,
         )
         if result.returncode == 0:
-            return
+            break
```

Comment:
Nice catch :)

Reply:
Thanks :D It was actually a stupid change I made just before creating the PR without testing it